api-admin/src/main/java/bio/terra/pearl/api/admin/service/notifications/SendgridActivityService.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.pearl.api.admin.service.notifications;
+
+import bio.terra.pearl.core.dao.notification.SendgridEventDao;
+import bio.terra.pearl.core.model.notification.SendgridEvent;
+import bio.terra.pearl.core.service.notification.email.SendgridClient;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
top level comment on this class would be helpful too
Should this be called "SendgridEventService" since it's corresponding to the SendgridEvent model?